### PR TITLE
Re-write docs for `globus-collection` format

### PR DESCRIPTION
**Pull request by jakeglobus**

### CODE REVIEW

File: changelog.d/20220224_133151_jake_globus_collection.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.. Bugfixes
+.. --------
+..
+.. - A bullet item for the Bugfixes category.
+..
+.. Documentation
+.. -------------
+..
+.. - A bullet item for the Documentation category.
+..
```
Comment (kurtmckee) [inline changelog.d/20220224_133151_jake_globus_collection.rst:19]: The leading `.. ` acts as a single-line comment. Please remove the comment lines and uncomment this section so it looks something like:

```rst
Documentation
-------------

- Document the ``globus-collection`` schema format.
```

File: docs/source/authoring_flows.rst [hunk 2]
```diff
@@ -427,48 +427,86 @@ An example structure for an ``ExpressionEval`` state is as follows:
...
 
-.. image:: _static/images/globus-collection-id-ex1.png
-  :alt: Example of the input created by globus collection id format
+``globus-collection`` as a ``format`` in your ``input_schema`` will signal to the webapp to show a custom input field for searching for and selecting a Globus collection on the Guided tab when starting a Flow.
 
-``globus-collection-path``
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+This example input schema shows how to configure a basic Transfer flow using this format:
 
 .. code-block:: JSON
```
Comment (kurtmckee) [inline docs/source/authoring_flows.rst:445]: There may be value in adding line emphasis. Sphinx supports a code-block option called [emphasize-lines](https://www.sphinx-doc.org/en/1.4.9/markup/code.html#line-numbers) that might be helpful here:

```rst
.. code-block:: json
   :emphasize-lines: 6, 9-10, 13-19

   {
     "additionalProperties": false,
     ...
```

This example would highlight the `format` line, the `required` lines, and the "id" and "path" property definition lines.

File: docs/source/authoring_flows.rst [hunk 3]
```diff
@@ -427,48 +427,86 @@ An example structure for an ``ExpressionEval`` state is as follows:
...
+Important notes about the ``globus-collection`` format:
+-------------------------------------------------------
 
-.. image:: _static/images/globus-collection-path-ex1.png
-  :alt: Example of the input created by globus collection id format
+* The properties inside ``globus-collection`` must be named "id" and "path"
+* The ``required`` field inside ``globus-collection`` determines behavior of the component as follows:
+   * If both "id" and "path" are required, the component will display and require both collection and path inputs
+   * If only "id" is required, the component will only display the Collection input
+   * If only "path" is required, the component will display both inputs but only the path field will be required. The collection input is provided to allow browsing of that collections directory listing
```
Comment (kurtmckee) [inline docs/source/authoring_flows.rst:508]:
```diff
- browsing of that collections directory
+ browsing of that collection's directory
```

File: docs/source/authoring_flows.rst [hunk 4]
```diff
@@ -427,48 +427,86 @@ An example structure for an ``ExpressionEval`` state is as follows:
...
+* This format is used to provide a UI component for `Globus web app`_ and will not substantively effect Flow usage from the Automate CLI or programmatic access
```
Comment (kurtmckee) [inline docs/source/authoring_flows.rst:509]:
```diff
- will not substantively effect Flow usage
+ will not substantively affect Flow usage
```

**Review by kurtmckee — CHANGES_REQUESTED** [review]
Looks good! I've left some suggestions about formatting, and some requests to fix some typos.

File: docs/source/authoring_flows.rst (hunk 4 above)
Comment (jakeglobus) [reply docs/source/authoring_flows.rst:509]: I'm forever confusing those two, thanks!

**Comment by jakeglobus** [comment]
All set on those, thanks Kurt

File: changelog.d/20220224_133151_jake_globus_collection.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+.. Bugfixes
+.. --------
+..
+.. - A bullet item for the Bugfixes category.
+..
+ Documentation
+.. -------------
+..
+- Documentation and examples for the ``globus-collection`` input schema format.
+..
```
Comment (kurtmckee) [inline changelog.d/20220224_133151_jake_globus_collection.rst:19]: "Documentation" has a leading space, and the next two lines still have the `dot-dot-space` prefix that makes them comments.

I think this won't get picked up correctly when the fragments are compiled together. It should match this format:

```rst
Documentation
-------------

- Documentation and examples for the ``globus-collection`` input schema format.
```

**Review by kurtmckee — CHANGES_REQUESTED** [review]

File: changelog.d/20220224_133151_jake_globus_collection.rst (hunk 5 above)
Comment (jakeglobus) [reply changelog.d/20220224_133151_jake_globus_collection.rst]: Okay I think I got it this time, thanks for your help Kurt

**Review by kurtmckee — APPROVED** [review]

File: changelog.d/20220224_133151_jake_globus_collection.rst (hunk 5 above)
Comment (kurtmckee) [reply changelog.d/20220224_133151_jake_globus_collection.rst]: NP, thanks for all your work on the new features and the docs!!

**Pull request merged**